lib/core/Cargo.toml

```diff
@@ -49,6 +49,8 @@ hex = "0.4"
 reqwest = { version = "=0.11.20", features = ["json"] }
 electrum-client = { version = "0.19.0" }
 zbase32 = "0.1.2"
+x509-parser = { version = "0.16.0", features = ["verify"] }
```

Comment:
It also works without the `verify` feature:
```toml
x509-parser = "0.16.0"
```

This keeps the `Cargo.lock`s and the binary size smaller.